router/core/src/main/scala/io/buoyant/router/ClassifiedRetries.scala

```diff
@@ -1,7 +1,8 @@
 package io.buoyant.router
 
 import com.twitter.finagle.{Stack, Stackable, ServiceFactory, param}
-import com.twitter.finagle.service._
+import com.twitter.finagle.buoyant.RetryFilter
```

Comment:
Hmm, if I revert this then nothing will be using the RetryFilter class that I'm adding in com.twitter.finagle.buoyant.